build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.0.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.88.0'
+    gutenbergMobileVersion = '5459-1e82ee9edc7a4fa5058fadfdb8da6735b94cf933'
```

Comment:
@SiobhyB This reference is no longer valid as the commits in the GB-mobile PR have been recreated due to a rebase ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5459#event-8502143957)). The last commit SHA that could be used here is: `81826df1f59b0002c77e7bc6a0dc4356fed2b273`.